pom.xml

```diff
@@ -22,7 +22,6 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-jpa</artifactId>
 		</dependency>
-
```

Comment:
Unnecessary modification - do not apply changes to files not related to your task.

Reply:
Not fixed